compiler/record-minmax/src/MinMaxObserver.cpp

```diff
@@ -61,7 +54,21 @@ void MinMaxObserver::postTensorWrite(const luci::CircleNode *node,
 
   // Only support recording of float32 values
   if (tensor->element_type() != DataType::FLOAT32)
-    throw std::runtime_error("Tensor's data type is not float");
+  {
+    // Argmax and Cast are exceptions we can process in backends
+    // Output of arg_max is the index of the largest value across axes of a tensor
+    // Cast converts float to int
```

Comment:
Cast is a more general operator. I think the comment can be revised (or removed?).

In fact, there are three Ops (ArgMax, Cast, Reshape) for now, and I think more Ops will be added to this list. I think explanation for each operator is not necessary.

Reply:
Comments seems good information to me :)
Maybe move to below each `case luci::CircleOpCode::XXX` line ?

Reply:
> Cast is a more general operator. I think the comment can be revised (or removed?).

You are right, when I wrote this I had just "int->float" and "float->int" cases in mind =_=